db/schema.rb

```diff
@@ -15,6 +15,19 @@
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "carriers", force: :cascade do |t|
+    t.integer "item_id"
+    t.string "name"
+    t.string "manufacturer"
+    t.string "model"
+    t.string "color"
+    t.integer "size"
+    t.integer "location_id"
+    t.integer "default_loan_length", default: 30
```

Comment:
Let's denote the unit of measure here, being `days`

Reply:
_days added for loan length